docs/google-pay-setup.md

```
@@ -9,6 +9,27 @@ Follow this checklist to enable Google Pay for the Android implementation of `@c
 - Android Studio Hedgehog (or newer) with the latest Android SDK tools.
 - Test devices running Google Play services.
 
+### Supported card networks
+For `allowedCardNetworks` options, the possible values are:
+- `AMEX`: American Express card network.
+- `DISCOVER`: Discover card network.
+- `ELECTRON`: Visa's Electron card network.
+   - Note that this option can only be set when `transactionInfo.countryCode` is set to `"BR"`, and `allowedCardNetworks` must also contain `VISA`
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 3

(MD007, ul-indent)